src/reflect/scala/reflect/internal/StdNames.scala

```diff
@@ -88,14 +88,12 @@ trait StdNames {
 
   abstract class CommonNames extends NamesApi {
     type NameType >: Null <: Name
-    // Masking some implicits so as to allow our targeted => NameType.
-    protected val stringToTermName = null
-    protected val stringToTypeName = null
-    protected implicit def createNameType(name: String): NameType
+    protected def createNameType(name: String): NameType
```

Comment:
Could we make this local method still implicit for a smaller diff?

Reply:
Unfortunately not. The ambiguous implicit stuff happens in this class if I keep `createNameType` as implicit even locally.

Reply:
Could it perhaps be called `def nameType`, to make it look like a "tag" or wrapper case class creation?